breacharbiter.go

```diff
@@ -87,7 +103,31 @@ func (b *breachArbiter) Start() error {
 
 	brarLog.Tracef("Starting breach arbiter")
 
-	// First we need to query that database state for all currently active
+	// We load any pending retributions from the database. For each retribution
```

Comment:
In addition to this check, the `breachArbiter` also needs to examine the channels on disk for thos that have been partially closed with a `channeldb.ClosureType` of `channeldb.BreachClose`. This is required as it's possible for the following series of events to take place: a channel is breached, the `breachObserver` detects it and updates the state on disk, but there's a crash before the message can be sent to the main `contractObserver` goroutine.